client/src/components/Chat/ChatHeader.js

```diff
@@ -1,34 +1,129 @@
-import React from 'react';
+import React, {useState} from 'react';
+import { makeStyles, withStyles } from '@material-ui/core/styles';
 import Grid from '@material-ui/core/Grid';
 import Avatar from '@material-ui/core/Avatar';
 import Typography from '@material-ui/core/Typography';
+import Switch from '@material-ui/core/Switch';
+import Icon from '@material-ui/core/Icon';
+import { deepOrange, deepPurple } from '@material-ui/core/colors';
+
+import {useAuth} from '../../context/auth-context';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: 'flex',
+    '& > *': {
+      margin: theme.spacing(1),
+    },
+  },
+  orange: {
+    color: theme.palette.getContrastText(deepOrange[500]),
+    backgroundColor: deepOrange[500],
+  },
+  purple: {
+    color: theme.palette.getContrastText(deepPurple[500]),
+    backgroundColor: deepPurple[500],
+  },
+  sectionHeader: {
+    flexBasis: '70px',
+    flexShrink: '0',
+    backgroundColor: '#fff',
+    marginBottom: '5px',
+    padding: '18px',
+  },
+}));
+
+const AntSwitch = withStyles((theme) => ({
+  root: {
+    width: 28,
+    height: 16,
+    padding: 0,
+    display: 'flex',
+  },
+  switchBase: {
+    padding: 2,
+    color: theme.palette.grey[500],
+    '&$checked': {
+      transform: 'translateX(12px)',
+      color: theme.palette.common.white,
+      '& + $track': {
+        opacity: 1,
+        backgroundColor: theme.palette.primary.main,
+        borderColor: theme.palette.primary.main,
+      },
+    },
+  },
+  thumb: {
+    width: 12,
+    height: 12,
+    boxShadow: 'none',
+  },
+  track: {
+    border: `1px solid ${theme.palette.grey[500]}`,
+    borderRadius: 16 / 2,
+    opacity: 1,
+    backgroundColor: theme.palette.common.white,
+  },
+  checked: {},
+}))(Switch);
 
 const ChatHeader = props => {
-  const {selectedContact} = props;
+  const {selectedContacts} = props;
+  const classes = useStyles();
+  const [state, setState] = React.useState({
```

Comment:
```suggestion
  const [state, setState] = useState({
```

Reply:
hi @bonnieli do you mind if I make these changes in my current working branch, #65 ?

Reply:
@bonnieli I made both changes in my working branch, #65 and pushed to the open pr. I still have some work to do in that pr.